src/locales/ua/product.json

```diff
@@ -0,0 +1,76 @@
+{
+    "imgAltInfo": "Фото продукту",
+    "imgsViewer" : {
+        "prev": "Попередній",
+        "next": "Наступний",
+        "close": "Закрити"
+    },
+    "color": "Колір",
+    "pattern": "Візерунок",
+    "productPrice": "Ціна",
+    "productDescription": {
+        "description": "Опис продукту",
+        "mainMaterial": "Основна тканина",
+        "innerMaterial": "Внутрішній матеріал",
+        "bottomMaterial": "Матеріал низу",
+        "strapLengthInCm": "Довжина лямок(см)"
+    },
+    "weight": {
+        "weightLabel": "Вага(кг)",
+        "volumeLabel": "Об'єм(л)"
+    },
+    "priceFrom": "від",
+    "sizeNotAvailable": "Товар недоступний",
+
+    "size": {
+        "carrentSize": "Розмір"
+    },
+    "pdpButtons": {
+        "cartButton": "В кошик",
+        "buyButton": "Купити зараз",
+        "submitButton": "Підтвердити",
+        "cancelButton": "Відхилити",
+        "leaveReply": "Залишити відповідь",
+        "inCart": "Перейти до кошика"
+    },
+    "tooltips": {
+        "addWishful": "Додати в бажане",
+        "removeWishful": "Видалити з бажаного",
+        "edit": "Редагувати",
+        "delete": "Видалити",
+        "unregisteredReply": "Вам потрібно увійти, щоб залишити відповідь",
+        "unregisteredComment": "Вам потрібно увійти щоб, залишити коментар",
+        "bought": "ористувач купив даний товар",
+        "feedbackComment": "Ваш коментар не підтверджений адміністратором",
+        "feedbackReply": "Ваша відповідь не підтвердженна адміністратором",
+        "itemInCart": "Товар вже в кошику",
+        "itemInCartAlready": "Додати товар у кошик"
+    },
+    "toastMessage": {
+        "addedToWishList": "Додано до вподобань",
+        "removedFromWishList": "Видалено з вподобань",
+        "addedToCard": "Додано в кошик"
+    },
+    "similarItems": {
+        "title": "З цим гарно пасує"
+    },
+    "reply":{
+        "submit": "Відповісти",
+        "loadMore": "Більше відповідей",
+        "hideBtn": "Приховати відповіді",
+        "text": "Поділіться своїми враженнями з іншими",
+        "answers": "Кількість відповідей: "
+    },
+    "comments": {
+        "title": "Відгуки",
+        "firstName": "Ім'я",
```

Comment:
A lot of translations are in common or must be moved there. Check this in all your translations and refactor, please.